pywell/entry_points.py

```diff
@@ -1,3 +1,24 @@
+from pywell.secrets_manager import get_secret
+import datetime
```

Comment:
I added this because it appears to be used in the `json_serial` function and my IDE couldn't find it. I'm not sure if the function was working before, and if so, how.